test/integration/http_integration.cc

```diff
@@ -254,11 +254,24 @@ IntegrationCodecClientPtr HttpIntegrationTest::makeRawHttpConnection(
         envoy::config::core::v3::Http2ProtocolOptions());
     http2_options.value().set_allow_connect(true);
     http2_options.value().set_allow_metadata(true);
-  } else if (http2_options.value().has_override_stream_error_on_invalid_http_message()) {
-    http3_options.mutable_override_stream_error_on_invalid_http_message()->set_value(
-        http2_options.value().override_stream_error_on_invalid_http_message().value());
-  } else if (http2_options.value().stream_error_on_invalid_http_messaging()) {
-    http3_options.mutable_override_stream_error_on_invalid_http_message()->set_value(true);
+#ifdef ENVOY_ENABLE_QUIC
+  } else {
```

Comment:
I'm fine doing this inline but I wonder if it should be a static config utility function.  WDYT?